test/test_op_normalization.py

```diff
@@ -0,0 +1,111 @@
+import torch
+from torch.testing import FileCheck
+
+from torch.testing._internal.common_utils import \
+    (run_tests)
+from torch.testing._internal.jit_utils import JitTestCase
+from torch.testing._internal.common_device_type import \
+    (instantiate_device_type_tests, skipCPUIfNoLapack, skipCUDAIfNoMagma)
+
+# Information for generating an alias test
+# NOTE: ending the alias_name with an underscore will interpret the test
+#   as the test for an inplace method of that name
+class AliasInfo(object):
+    __slots__ = ['alias_name', 'alias_op', 'original_name', 'input', 'args', 'decorators']
+
+    def __init__(self,
+                 alias_name,  # the name of the alias
+                 alias_op,  # the aliased op
+                 original_name,  # the name of the original function
+                 input,  # the first tensor argument to the op
+                 *,
+                 args=(),  # tuple of additional positional arguments
+                 decorators=()):  # decorators to apply to the test
+        self.alias_name = alias_name
+        self.alias_op = alias_op
+        self.original_name = original_name
+        self.input = input
+        self.args = args
+        self.decorators = decorators
+
+alias_infos = (
+    AliasInfo('absolute', torch.absolute, 'abs',
```

Comment:
I think it's kind of nice to reuse the op registry instead of creating a new one...

Reply:
It is! But check this out: https://github.com/pytorch/pytorch/pull/41662.

So now we can update this to use the fancy new op registry.